docs/ref/contrib/postgres/search.txt

```diff
@@ -189,3 +189,59 @@ if it were an annotated ``SearchVector``::
     [<Entry: Cheese on Toast recipes>, <Entry: Pizza recipes>]
 
 .. _PostgreSQL documentation: http://www.postgresql.org/docs/current/static/textsearch-features.html#TEXTSEARCH-UPDATE-TRIGGERS
+
+
+``TrigramSimilarity``
+=====================
+
+.. class:: TrigramSimilarity(expression, expression, **extra)
```

Comment:
Looks good – only things I could see were in the text around line 90:
* Extraneous "to" in "Another approach to searching is to trigram similarity."
* Missing "of" in "you can use a couple other expressions.
* The postgresql.org link I think is missing the end `_ 
Thanks :)